src/all.less

```
@@ -1,2 +1,3 @@
+@import "~esn-frontend-inbox/src/esn.inbox.libs/app/inbox.libs.less";
```

I commented in another PR for the naming convention
```suggestion
@import "~esn-frontend-inbox/src/esn.inbox.libs/app/app.less";
```